### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PRIYANKjakharia**
correction of a few grammatical errors and typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Current State of the Project
 
 **Important notice**: At the moment, "gameplay" is basically non-functional.
 We're implementing the internal game simulation (how units even do anything) with simplicity and extensibility in mind, so we had to get rid of the temporary (but kind of working) previous version.
-With these changes we can (finally) actually make use of our converted asset packs and our nyan API!
+With these changes, we can (finally) actually make use of our converted asset packs and our Nyan API!
```
Comment (heinezen) [inline README.md:82]:
```suggestion
With these changes, we can (finally) actually make use of our converted asset packs and our nyan API!
```

nyan is intended to use a lowercase name

File: README.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ just have to ask and we'll find something. Alternatively, lurking is also allowe
 
 ----
 
-Cheers, happy hecking!
+Cheers, happy hacking!
```
Comment (heinezen) [inline README.md:189]:
```suggestion
Cheers, happy hecking!
```

The mispelling is intentional ;)

**Review by heinezen — COMMENTED** [review]
Thanks a lot. Can you fix the two things I mentioned in the comments?

**Comment by PRIYANKjakharia** [comment]
Done thanks for cooperating.

**Review by heinezen — APPROVED** [review]

**Comment by heinezen** [comment]
Indeed, thanks for your help!

**Pull request merged**